### DISCUSSION TITLE
Welcome to LingmoOS Discussions!

### DISCUSSION THREAD

**Discussion by lingmo-dream** [LingmoOS/Community#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dumitru88888** [comment]
When will the new version of the operating system be released?

**Comment by Dumitru88888** [comment]
You need to create your own installer in the operating system.

**Comment by Dumitru88888** [comment]
You need to develop your own system monitor.

**Comment by habosbaboskobanyai1** [comment]
Help me build 3.0 Helium

**Comment by Dumitru88888** [comment]
You need to create your own program in the same style as the operating system, I mean the operating system for disk editing. You need to create it to be beautiful, like in the system.

**Comment by Dumitru88888** [comment]
Need to create your own installer please, nice as a system.

**Comment by Dumitru88888** [comment]
You can as an idea create a program to uninstall applications in a style like the operating system.

**Comment by Dumitru88888** [comment]
The app store needs to be honed to be as much in design as the operating system.

**Comment by Dumitru88888** [comment]
We need to add a palette of colors.

**Comment by Dumitru88888** [comment]
Copy computer specifications in one click.

**Comment by Dumitru88888** [comment]
You need to add folders to the Launcher.
